icu4c/source/i18n/uspoof.cpp

```diff
@@ -546,7 +546,9 @@ uspoof_checkUnicodeString(const USpoofChecker *sc,
     return uspoof_check2UnicodeString(sc, id, NULL, status);
 }
 
-static int32_t checkImpl(const SpoofImpl* This, const UnicodeString& id, CheckResult* checkResult, UErrorCode* status) {
+namespace {
```

Comment:
Agh, GitHub won't let me comment on lines that aren't in the diff.
Thanks for this change!
Though, I think the intent of the ticket was to use anonymous namespaces for **all** of the file-level `static` functions, not just the `checkImpl` one. (Ex: functions like `uspoof_cleanup`, `initializeStatics`, etc..)

Reply:
Oops, that's my bad. I'll update this in another commit, thanks!